packages/cms/pages/assests/asset-manager/asset-manager.tsx

```diff
@@ -223,33 +241,33 @@ export const AssetManager: FunctionComponent = () => {
   const closeDestroyModal = () => setIsDestroyModalVisible(false)
   const showDestroyModal = () => setIsDestroyModalVisible(true)
 
-  const [assets, setAssets] = useState([])
+  const [assets, setAssets] = useState<any>([])
   const [active, setActive] = useState<AssetData>()
 
   const [activePage, setActivePage] = useState(0)
   const [pageCount, setPageCount] = useState(0)
   const [isPopoverOpen, setIsPopoverOpen] = useState(false)
   const [perPage, setPerPage] = useState(10)
-  const [loading, setLoading] = useState(false)
+  const [isLoading, setIsLoading] = useState(false)
   const [search, setsearch] = useState('')
 
-  useEffect(() => {
-    setLoading(true)
-    const fetchFiles = async () => {
-      const params = {
-        page: activePage + 1,
-        perPage,
-        ...(search && { search })
-      }
-      const [data, error] = await window.Tensei.api.get('files', { params })
-      if (!error) {
-        setLoading(false)
-        setAssets(data?.data.data)
-        setPageCount(data?.data.meta.pageCount)
-        return
-      }
-      setLoading(false)
+  const fetchFiles = async () => {
+    const params = {
+      page: activePage + 1,
+      perPage,
+      ...(search && { search })
+    }
+    const [data, error] = await window.Tensei.api.get('files', { params })
```

Comment:
The data variable is technically a response. Should be a little easier to read as `response.data.data` instead of `data.data.data`